src/pybind/ceph_argparse.py

```diff
@@ -1193,7 +1199,7 @@ def validate_command(sigdict, args, verbose=False):
             print("Invalid command:", ex, file=sys.stderr)
             print(concise_sig(sig), ': ', cmd['help'], file=sys.stderr)
     else:
-        bestcmds = bestcmds[:10]
+        bestcmds = filter(lambda c: c.get('flags', 0) & ~(FLAG_DEPRECATED | FLAG_HIDDEN), bestcmds)[:10]
```

Comment:
this does not look right, see 

https://github.com/ceph/ceph/blob/b145cf0a0470638379ae33614f63088f94d7251e/src/mon/MonCommand.h#L27

so a valid command might has a flag of `FLAG_NONE`, and i think python takes `0` as `False` in the `filter()` function. so probably we should put something like
```py
bestcmds = [c for c in bestcmds[:10] if not c.get('flags', 0) & (FLAG_DEPRECATED | FLAG_HIDDEN)]
```
in this change, i
1. use list comprehension instead `filter()` for better readablity
1. put `[:10]` in the loop, for slightly better performance.
1. add `not`. thanks god, neither `FLAG_DEPRECATED` nor `FLAG_HIDDEN` is 0. =)

Reply:
As always, thanks for the meticulous review Kefu! You're quite right. I've made this change but kept the `[:10]` outside the loop as otherwise hidden/deprecated commands will reduce the number of suggestions.